### DISCUSSION TITLE
Set meta values through code?

### DISCUSSION THREAD

**Discussion by ianleeder** [patrickdemooij9/SeoToolkit.Umbraco#140] [Q&A]
We're doing a bulk-import of pages for site creation, and I've already sorted out creating content nodes using [`ContentService`](https://apidocs.umbraco.com/v9/csharp/api/Umbraco.Cms.Core.Services.Implement.ContentService.html).  I'm trying to work out if it's possible to import with SEO data set, I can't see anything in the documentation about this. I've looked through the `IContent.Properties` returned, and can't see anything helpful.  Is it possible to insert SEO data when creating a node through `ContentService` (or some other way)?

**Comment by patrickdemooij9** [comment]
That is a good question. Do you mean the data on the document type page or the one on the content page? I will have to take a look at this and then document it. It should be possible, but I'll have a look

**Comment by ianleeder** [comment]
In my case, I’m wanting to set it on content when importing.
